### PR TITLE
style: organised nested css

### PR DESCRIPTION
**Pull request by aurelienLavanchy**

### CODE REVIEW

File: client/src/components/navbar/NavBar.css [hunk 1]
```diff
@@ -35,31 +35,33 @@
...
-    display: block;
-    background-color: var(--main-color-gray);
-    position: relative;
-    top: 0;
-    width: 2.5rem;
-    height: 4px;
-    border-radius: 10px;
-    transition: all 300ms ease-in-out;
-  }
+    span {
```
Comment (MichaelLELU) [inline client/src/components/navbar/NavBar.css:40]: you do css on a semantic balise ?

**Review by MichaelLELU — COMMENTED** [review]
what is &: and &. in css in go check (but thanks for all you do )

File: client/src/components/navbar/NavBar.css (hunk 1 above)
Comment (aurelienLavanchy) [reply client/src/components/navbar/NavBar.css:40]: why not ?

**Comment by aurelienLavanchy** [comment]
> what is &: and &. in css in go check (but thanks for all you do )

you can find info on css nesting on [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_nesting/Using_CSS_nesting#combinators) about using `&`, `&.class` and more

**Review by MichaelLELU — APPROVED** [review]

**Pull request merged**